doc/source/examining/loading_data.rst

```diff
@@ -271,16 +271,18 @@ will be in code units.
 
 .. _loading-orion-data:
 
-BoxLib Data
+AMReX / BoxLib Data
 -----------
 
-yt has been tested with BoxLib data generated by Orion, Nyx, Maestro, Castro, 
-and WarpX. Currently it is cared for by a combination of Andrew Myers, Chris
-Malone, Matthew Turk, and Mike Zingale.
+AMReX and BoxLib share a frontend (currently named `boxlib`), since
+the file format is identical.  yt has been tested with AMReX/BoxLib
```

Comment:
Nit - the particle file format is slightly different in AMReX. In the old format, there wasn't enough information in the header to fully specify how to read the particles, and we had to make application-specific assumptions in the BoxLib code frontends. For example, you had to know whether the code in question wrote the integer ids to disk to interpret the binary data correctly. Now more metadata is stored in the headers. There is also the option to write additional particle integer fields, if any are defined. 

I don't think there's any need to get in to this here, but maybe I'd say they're **nearly** identical.